### ISSUE TITLE
Chronos: add learning rate scheduler creator in pytorch models

### ISSUE THREAD

**Issue by shane-huang** [intel-analytics/BigDL#4835]
Besides optimizer_creator , loss_creator and model_creator in each model. we should also have a `scheduler_creator` or let `optimizer creator` return `(schedular, optimizer)` instead of `optimizer`.
